content/3_Silver/Intro_Tree.mdx

```diff
@@ -352,10 +361,11 @@ int main()
 
 ## Problems
 
-<Problems problems={problems.tree} />
-
-<Info title="Wizard's Tour">
+<Info>
 
-Try solving this problem on a tree first!
+For the last two problems, try solving them on a tree first.
 
 </Info>
```

Comment:
I think this is unnecessary info. It's located at the beginning of the problemset but is irrelevant for the majority of the problems.

I say we either move it to the end of the pset, use footnotes, or delete it.

Reply:
And it might confuse people who just read the module and don't know about spanning trees. On that note, should we define spanning tree in the module?

Reply:
maybe, that can be a separate PR